lib/matplotlib/tests/test_contour.py

```diff
@@ -309,3 +309,53 @@ def test_contourf_symmetric_locator():
     locator = plt.MaxNLocator(nbins=4, symmetric=True)
     cs = plt.contourf(z, locator=locator)
     assert_array_almost_equal(cs.levels, np.linspace(-12, 12, 5))
+
+
+def test_contour_1x1_array():
```

Comment:
This test might need a cleanup decorator, because while the `plt.contour()` calls all fail, figures and axes are still produced prior to throwing the error.